docs/source/examples.select_fps.ipynb

```diff
@@ -0,0 +1,102 @@
+    "\n",
+    "In this example, we show how FPS can be used for selection of \"n=8\" datapoints from an MD trajectory by comparing \"average SOAP descriptors\" of all path configurations. This is done in two steps: \n",
+    "1. Assigning  a global descriptor for each configuration in the trajectory followed by \n",
+    "2. A call of the *greedy_fps_conf_global* function \n",
+    "\n",
+    "To assign a per-config descriptor we calculate the average SOAP vector for every frame in the MD trajectory. The greedy-FPS algorithm would use them to measure similarities across the datapoints and select 10 unique structures. \n",
+    "\n",
+    "Overall this requires two input files: the database and the descriptors (\"md.traj\" and \"params.yaml\")\n",
+    "\n",
+    "Tip: params.yaml can either be self-written or automatically generated from a \"Univeral_SOAP-template\" processed by multi-stage gap fit (see wfl.fit.gap.multistage)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a209cdf-1153-4e7d-bedb-dcd8cea04116",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wfl, os, sys, yaml\n",
+    "from wfl.configset import ConfigSet, OutputSpec\n",
+    "from wfl.descriptors.quippy import calc as calc_descriptors\n",
+    "from wfl.select.by_descriptor import greedy_fps_conf_global as get_fps\n",
+    "\n",
+    "work_dir = os.path.join(os.path.dirname(wfl.__file__), \"../docs/source/examples_files/select_fps\") # Location of I/O files\n",
+    "\n",
+    "# Step 1: Assign descriptors to the database\n",
+    "md        = ConfigSet(os.path.join(work_dir, \"md.traj\"))\n",
+    "md_desc   = OutputSpec(files=os.path.join(work_dir, \"md_desc.xyz\"))\n",
+    "\n",
+    "with open(os.path.join(work_dir, 'params.yaml'), 'r') as foo:\n",
+    "    desc_dict = yaml.safe_load(foo)\n",
+    "desc_dicts = [d for d in desc_dict if 'soap' in d.keys()] # filtering out only SOAP descriptor\n",
+    "per_atom = True\n",
+    "for param in desc_dicts:\n",
+    "    if 'average' not in param.keys():\n",
+    "        param['average']= True # to create global (per-conf) descriptors instead of local (per-atom)\n",
+    "        per_atom = False\n",
+    "md_desc = calc_descriptors(inputs=md, outputs=md_desc, descs=desc_dicts, key='desc', per_atom=per_atom)\n",
+    "\n",
+    "# Step 2: Sampling\n",
+    "fps      = OutputSpec(files=os.path.join(work_dir, \"out_fps.xyz\"))\n",
+    "nsamples = 8\n",
+    "get_fps(inputs=md_desc, outputs=fps, num=nsamples, at_descs_info_key='desc', keep_descriptor_info=False)"
```

Comment:
Just to keep the workflow principles, how about `selected_configs = get_fps(....`?